File: TesteB3.Server/Domain/Model/TItuloModel.cs
using System.Diagnostics.CodeAnalysis;

namespace TesteB3.Server.Domain.Model
{
    [ExcludeFromCodeCoverage]
    public class TituloModel
    {
        public string totalLiquido { get; set; }
        public decimal valor { get; set; }
        public int prazo { get; set; }
        public string totalBruto { get; set; }
    }
}

File: testeb3.client/src/app/titulo.model.ts
export class TituloModel {
  public valor: number;
  public prazo: number;
  public totalBruto: string;
  public totalLiquido: string;

  constructor() {
    this.prazo = 0;
    this.valor = 0;
    this.totalBruto = "0.00";
    this.totalLiquido = "0.00";
  }
      
}
